database/src/migrations/release.0.5/restoration.sql

```diff
@@ -11,19 +11,19 @@
 --
 
 CREATE TABLE administrative_activity(
-    administrative_activity_id                integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
-    administrative_activity_status_type_id    integer           NOT NULL,
-    administrative_activity_type_id           integer           NOT NULL,
-    reported_system_user_id                   integer           NOT NULL,
+    administrative_activity_id                integer          GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    administrative_activity_status_type_id    integer          NOT NULL,
+    administrative_activity_type_id           integer          NOT NULL,
+    reported_system_user_id                   integer          NOT NULL,
     assigned_system_user_id                   integer,
     description                               varchar(3000),
-    data                                      json,
+    data                                      jsonb,
     notes                                     varchar(3000),
-    create_date                               timestamptz(6)    DEFAULT now() NOT NULL,
-    create_user                               integer           NOT NULL,
-    update_date                               timestamptz(6),
-    update_user                               integer,
-    revision_count                            integer           DEFAULT 0 NOT NULL,
+    create_date                               timestamp        DEFAULT now() NOT NULL,

```

Comment:
I remember we changed to `timestampz(6)` a while back because we were having date timezone issues.  Just wondering if this change back to just `timestamp` will bring that issue back? We'll have to double check, just in case.

Reply:
thanks for identifying these issues... move to a new machine did not bring over all of the necessary config.